tests/integration_tests/build/test_binary_size.py

```diff
@@ -11,10 +11,10 @@
 MACHINE = platform.machine()
 """ Platform definition used to select the correct size target"""
 
-FC_BINARY_SIZE_TARGET = 3013464 if MACHINE == "x86_64" else 3251464
+FC_BINARY_SIZE_TARGET = 3200000 if MACHINE == "x86_64" else 3251464
 """Firecracker target binary size in bytes"""
 
-FC_BINARY_SIZE_LIMIT = 3200000 if MACHINE == "x86_64" else 3400000
+FC_BINARY_SIZE_LIMIT = 3500000 if MACHINE == "x86_64" else 3400000
```

Comment:
The `3500000` limit is pretty big comparing with the current binary size (`3194680` from what I've seen when running this test and forcing it it to fail).
Also, more of a nit, but can we update the `FC_BINARY_SIZE_TARGET` with a more accurate value? I think it's nice to modify with precise values when the test requires to update these values (it keeps somehow the history clearer).